subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/DisambiguateArtifactTransformIntegrationTest.groovy

```diff
@@ -43,7 +43,6 @@ public class MyClass {
 
         buildFile << """
 def artifactType = Attribute.of('artifactType', String)
-//attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage.class, Usage.JAVA_API))
```

Comment:
if it's intentional, remove the line, otherwise it's unclear if it's a leftover of a test.

Reply:
It was a left over and this removes it